view/dust.js

```diff
@@ -40,7 +40,7 @@ exports.create = function (config) {
         global = context.global;
         locals = context.get('context');
         locality = util.localityFromLocals(locals);
-        props = res.resolve(name, locality).file || i18n.contentPath;
```

Comment:
In the case that the content supplied is missing we will just pass along `undefined` to signal that we couldn't find any properties file. This only matters if we call `{@pre}` somewhere in the file. Dust files don't *need* have content, so we preserve this and then if `localizr` is called to get the content we can say there with confidence: "Someone is asking for content and we can't find it"